src/EditorFeatures/Test/SolutionCrawler/WorkCoordinatorTests.cs

```diff
@@ -252,6 +252,23 @@ public async Task Project_AssemblyName_Change()
             }
         }
 
+        [Fact]
+        public async Task Project_DefaultNamespace_Change()
+        {
+            using (var workspace = new WorkCoordinatorWorkspace(SolutionCrawler))
+            {
+                var solutionInfo = GetInitialSolutionInfo_2Projects_10Documents(workspace);
+                workspace.OnSolutionAdded(solutionInfo);
+                await WaitWaiterAsync(workspace.ExportProvider);
+
+                var project = workspace.CurrentSolution.Projects.First(p => p.Name == "P1").WithDefaultNamespace("newNamespace");
+                var worker = await ExecuteOperation(workspace, w => w.ChangeProject(project.Id, project.Solution));
+
+                Assert.Equal(5, worker.SyntaxDocumentIds.Count);
+                Assert.Equal(5, worker.DocumentIds.Count);
```

Comment:
it seems strange that this test isn't actually validating the namespace change.  I'm not sure what it's actually validating.  it just seems to be looking at SyntaxIds and DocumetnIds... but i'm not sure why we'd care about those when changing a namespace.

Reply:
This test has nothing to do with changing namespace. It's just testing the behavior of solution crawler after changing "default namespace" for a project.